lib/MusicBrainz/Server/Entity/LinkType.pm

```diff
@@ -149,6 +151,7 @@ around TO_JSON => sub {
     $json->{reverse_link_phrase} = $self->reverse_link_phrase;
     $json->{type0} = $self->entity0_type;
     $json->{type1} = $self->entity1_type;
+    @children ? ($json->{children} = \@children) : (),
```

Comment:
Should probably be a statement instead of list element, seems to be intended as:

```suggestion
    $json->{children} = \@children if @children;
```